cmake/Platform/Windows/Packaging/Bootstrapper.wxs

```diff
@@ -5,7 +5,7 @@
 <Wix xmlns="http://schemas.microsoft.com/wix/2006/wi"
     xmlns:bal="http://schemas.microsoft.com/wix/BalExtension">
 
-    <Bundle Name="$(var.CPACK_PACKAGE_NAME) [Developer Preview]"
```

Comment:
We probably don't want this change going back to development

Reply:
All good calls, will fix.